controllers/postulants.js

```diff
@@ -0,0 +1,206 @@
+const fs = require('fs');
+const { stringify } = require('querystring');
+const Postulants = require('../data/postulants.json');
+
+const getPostulants = (req, res) => {
+  res.status(200).json(Postulants);
+}
+
+const addPostulant = (req, res) => {
+  //Convertion of field "open_to_work" from "string" to "boolean"
+  const boolOpenToWork = req.query.open_to_work.toLowerCase() === 'true';
+
+  const newPostulant = {
+    id: (Postulants.length) + 1,
+    first_name: req.query.first_name,
+    last_name: req.query.last_name,
+    gender: req.query.gender,
+    date_of_birth: req.query.date_of_birth,
+    email: req.query.email,
+    phone: req.query.phone,
+    city: req.query.city,
+    state: req.query.state,
+    country: req.query.country,
+    elementary_school: {
+      name: req.query.elementary_school_name,
+      graduate_year: req.query.elementary_school_graduate_year,
+    },
+    high_school: {
+      name: req.query.high_school_name,
+      graduate_year: req.query.high_school_graduate_year,
+    },
+    junior_college: {
+      name: req.query.junior_college_name,
+      degree: req.query.junior_college_degree,
+      graduate_year: req.query.junior_college_graduate_year,
+    },
+    university: {
+      name: req.query.university_name,
+      degree: req.query.university_degree,
+      graduate_year: req.query.university_graduate_year,
+    },
+    open_to_work: boolOpenToWork,
+    work_experience: {
+      title: req.query.work_experience_title,
+      start: req.query.work_experience_start,
+      end: req.query.work_experience_end,
+      company: req.query.work_experience_company,
+      description: req.query.work_experience_description,
+    },
+    professional_training: {
+      description: req.query.professional_training_description,
+      year: req.query.professional_training_year,
+    },
+    languages: [req.query.languages0, req.query.languages1, req.query.languages2],
+    hobbies: req.query.hobbies,
+    family_members: {
+      name: req.query.family_members_name,
+      bond: req.query.family_members_bond
+    },
+    availability: {
+      from: req.query.availability_from,
+      to: req.query.availability_to,
+      day: 
+        [
+          req.query.availability_day1, req.query.availability_day2, req.query.availability_day3, 
+          req.query.availability_day4, req.query.availability_day5, req.query.availability_day6, 
+          req.query.availability_day7
+        ],
+    }
+  }
+  
+  Postulants.push(newPostulant);
+  fs.writeFile('./data/postulants.json', JSON.stringify(Postulants), (err) => {
+    if (err) {
+      res.status(400).json({msg: "Error. Check the data sended and try again."});
```

Comment:
Also, there's no need to return a message as a json format inside of an object. We could just a string instead:

```suggestion
        res.status(404).send("Error. Check the data sent and try again.");
```

Reply:
Noted! Now all the messages are a string. 👍